promptsource/promptsource.py

```diff
@@ -477,12 +485,16 @@ def get_infos(d_name):
                             elif updated_template_name == "":
                                 st.error("Need to provide a template name.")
                             else:
+                                # Parses state.answer_choices
+                                updated_answer_choices = [x.strip() for x in state.answer_choices.split(",")]
```

Comment:
Not sure if this matters but this means that none of the answer choices can have commas in them. I could imagine targets of the form `["No, it's not", "Yes, it is"]`. But, if this is a pain, we can just ignore.

Reply:
Good point. A few templates with automatically extracted answer choices already have commas, so we should pick something else. We could just use `|||` again.

Reply:
We could probably use tabs, but I think tabs are hard to enter in browser interfaces. Maybe newlines? Or yeah, ||| is fine, I'm just worried it will be confusing to use it in two different contexts.

Reply:
Yeah, I think either of those are clunky if we want a single-line input. Pushed ||| for now, but very easy to change.